pylint/pyreverse/dot_printer.py

```diff
@@ -92,17 +92,22 @@ def _build_label_for_node(self, properties: NodeProperties) -> str:
 
         # Add class attributes
         attrs: list[str] = properties.attrs or []
-        attrs_string = r"\l".join(attr.replace("|", r"\|") for attr in attrs)
-        label = rf"{{{label}|{attrs_string}\l|"
+        attrs_string = r'<br ALIGN="LEFT"/>'.join(
```

Comment:
``<br ALIGN="LEFT"/>`` could be a class constant instead (and used at line 110 too).

Reply:
Added class constant :+1: